main.go

```diff
@@ -55,6 +59,9 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(mdbv1.AddToScheme(scheme))
+
+	utilruntime.Must(dbaas.DBaaSAddToScheme(scheme))
```

Comment:
Can we make this just AddToScheme instead of DBaaSAddToScheme so it matches the other packages/usages?

Reply:
Here are we need to register CRDs for two API groups, atlas.mongodb.com and dbaas.redhat.com. That's why two variables are used. Let me know if this can be enhanced further :)

Reply:
Yeah, I get that we need to add several packages to Scheme, I'm just wondering why it's `dbaas.DBaaSAddToScheme` instead of just `dbaas.AddToScheme` like the others. Is this still b/c of multiple groups? Do we have a few copies of this method so we had to prepend 'DBaaS' to the 'AddToScheme' part?

Reply:
That sounds good to me. I renamed it to AddToScheme.